pkg/cli/start_jemalloc.go

```diff
@@ -36,7 +36,11 @@ import (
 // //   /etc/malloc.conf and MALLOC_CONF can be safely set any time prior to
 // //   program invocation.
 // #ifdef __linux__
+// #  ifdef __s390x__
```

Comment:
Very minor nit: I could see an argument that the changes to this file might be useful as its own PR/commit as it's kind of independent of the rest of the changes, should probably be there even if it were determined this PR should be reverted, etc. Not gonna make a big deal out of that.

Reply:
I 100% see your point, and will split that kind of change into its own PR next time!